playground.ts

```diff
@@ -0,0 +1,27 @@
+import {HederaProvider} from "@ethersproject/providers";
+
+
+function encode(data: Uint8Array) :string {
+    return Buffer.from(data).toString("hex");
+}
+
+function toSolidityAddress(address: [number, number, number]) {
+    const buffer = new Uint8Array(20);
+    const view = new DataView(buffer.buffer, 0, 20);
+    const [shard, realm, num] = address;
+
+    view.setUint32(0, shard);
+    view.setUint32(8, realm);
+    view.setUint32(16, num);
+
+    return encode(buffer);
+}
+
+// main - run with ` ts-node playground.js `
+(async () => {
```

Comment:
I think that we can have 2 examples:

1. Example with `0x0000000000...98` (it is not 98 but you got the point)
2. Example with `hethers.utils.getAddress("0.0.98")` (converts it to eth address) + `provider.getBalance(ethAddress)`